tests/unit_tests/test_wave_03.py

```diff
@@ -130,8 +130,9 @@ def test_swap_items_from_their_empty_returns_false():
     nobodys_item = Item()
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
-
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    
+    assert result is False 
+    assert item_b in fatimah.inventory
```

Comment:
It's good that you check that `item_b` is still in `fatimah.inventory` to confirm that vendors' inventories weren't changed. 

It would make this test more robust to check that `item_a` and `item_c` are also still in Fatimah's inventory.